pkg/cmd/cli/cmd/importer/dockercompose.go

```diff
@@ -0,0 +1,182 @@
+package importer
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apimachinery/registered"
+	"k8s.io/kubernetes/pkg/kubectl"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	configcmd "github.com/openshift/origin/pkg/config/cmd"
+	"github.com/openshift/origin/pkg/generate/app"
+	appcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+	"github.com/openshift/origin/pkg/generate/dockercompose"
+)
+
+const (
+	dockerComposeLong = `
+Import a Docker Compose file as OpenShift objects
+
+Docker Compose files offer a container centric build and deploy pattern for simple applications.
+This command will transform a provided docker-compose.yml application into its OpenShift equivalent.
+During transformation fields in the compose syntax that are not relevant when running on top of
+a containerized platform will be ignored and a warning printed.
+
+The command will create objects unless you pass the -o yaml or --as-template flags to generate a
+configuration file for later use.`
+
+	dockerComposeExample = `  # Import a docker-compose.yml file into OpenShift
+  $ %[1]s docker-compose -f ./docker-compose.yml
+
+	# Turn a docker-compose.yml file into a template
```

Comment:
That is the thing I hate most about vs.code.

On Mon, May 9, 2016 at 1:54 PM, Cesar Wong <notifications@github.com> wrote:

> In pkg/cmd/cli/cmd/importer/dockercompose.go
> <https://github.com/openshift/origin/pull/7690#discussion_r62542496>:
>
> > +const (
> > +	dockerComposeLong = `
> > +Import a Docker Compose file as OpenShift objects
> > +
> > +Docker Compose files offer a container centric build and deploy pattern for simple applications.
> > +This command will transform a provided docker-compose.yml application into its OpenShift equivalent.
> > +During transformation fields in the compose syntax that are not relevant when running on top of
> > +a containerized platform will be ignored and a warning printed.
> > +
> > +The command will create objects unless you pass the -o yaml or --as-template flags to generate a
> > +configuration file for later use.`
> > +
> > +	dockerComposeExample = `  # Import a docker-compose.yml file into OpenShift
> > +  $ %[1]s docker-compose -f ./docker-compose.yml
> > +
> > +	# Turn a docker-compose.yml file into a template
>
> fix indent
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7690/files/41da1c779a29f6d9f95e64a1a35ce5283cddf0d1#r62542496>
>